src/server/scripts/Spells/spell_warrior.cpp

```diff
@@ -683,6 +685,45 @@ class spell_warr_second_wind_trigger : public SpellScriptLoader
         }
 };
 
+// 107570 - Storm Bolt
+class spell_warr_storm_bolt : public SpellScriptLoader
+{
+public:
+    spell_warr_storm_bolt() : SpellScriptLoader("spell_warr_storm_bolt") { }
+
+    class spell_warr_storm_bolt_SpellScript : public SpellScript
+    {
+        PrepareSpellScript(spell_warr_storm_bolt_SpellScript);
+
+        bool Validate(SpellInfo const* /*spellInfo*/) override
+        {
+            return ValidateSpellInfo
+            ({
+                SPELL_WARRIOR_STORM_BOLT,
```

Comment:
Don't validate the spell you are attaching the script to (107570), it is done by ObjectMgr already and the script is never created